plugins/MauticEmailMarketingBundle/Integration/EmailAbstractIntegration.php

```diff
@@ -60,6 +62,13 @@ public function getFormTheme()
         return 'MauticEmailMarketingBundle:FormTheme\EmailMarketing';
     }
 
+    /**
+     * Returns form type.
+     *
+     * @return string|null
+     */
+    abstract public function getFormType();
```

Comment:
This should be mentioned in [the GDoc](https://docs.google.com/document/d/1lHNK8hHQtvyWlz9OtfAoLupDVb5-QunQjq-HAGlAsS4/edit#heading=h.9dko8rvwv726) as well as it may break a 3rd party plugin

Reply:
Done.